README.md

```diff
@@ -182,7 +182,7 @@ metrics = require('metrics')
   Returns Histogram object.
   * `name` Collector name (string). Must be unique.
   * `help` (optional) Help description (string).
-  * `buckets` Histogram buckets (array of positive sorted numbers). `INF` bucket is added automatically. Default is 
+  * `buckets` Histogram buckets (array of positive sorted numbers). `INF` bucket is added automatically. Default is
```

Comment:
`an array of sorted positive numbers`, `Infinity bucket (INF) is appended automatically`